cli/migrate.md

```diff
@@ -1,14 +1,14 @@
 # Migration
-This section covers the steps to migrate your projects initialized using the Amplify CLI version ( < 0.2.0) which doesn't support multiple environments and team workflows. Environment and team workflow support is in beta and can be installed using the following command
+This section covers the steps to migrate your projects initialized using the Amplify CLI version (< 1.0.0) which doesn't support multiple environments and team workflows. Environment and team workflow support is now supported in as a part of the latest Amplify CLI version (>= 1.0.0) and can be installed using the following command:
```

Comment:
1. This section covers the steps to migrate your projects initialized using the Amplify CLI version -->
This section covers the steps to migrate your projects initialized using the older Amplify CLI versions (< 1.0.0)

"Environment and team workflow support is now supported in as a part of the latest Amplify CLI version (>= 1.0.0)" --> 
Environment and team workflow support is supported in Amplify CLI version (>= 1.0.0)